rhoas-cli-service-registry/README.adoc

```diff
@@ -97,6 +97,10 @@ This example creates a {registry} instance called `my-registry`.
 ----
```

Comment:
Line 95: Remove all of the subheadings that are shown immediately before a command input?
See also lines 129, 169, 200, 223, 247, 263, 307, 322, 343, 402, 410, 441, and 461.
(All of these line numbers refer to the original unedited file that's currently in the main branch.)

Reply:
I added back the code block titles.